service/application/src/main/java/com/solace/maas/ep/event/management/agent/config/SolaceConfiguration.java

```diff
@@ -65,6 +66,7 @@ public MessagingService messagingService() {
         log.info("Connecting to event portal using EMA client {}.", clientName);
         return MessagingService.builder(ConfigurationProfile.V1)
                 .fromProperties(vmrConfiguration)
+                .withReconnectionRetryStrategy(RetryStrategy.foreverRetry(15000))
```

Comment:
This is the real change here, the rest is just my formatter moving things around.